src/contracts/interfaces/IRewardsCoordinator.sol

```diff
@@ -143,11 +143,19 @@ interface IRewardsCoordinator {
         bytes32 indexed rewardsSubmissionHash,
         RewardsSubmission rewardsSubmission
     );
+    /// @notice emitted when a valid RewardsSubmission is created when rewardAllStakersAndOperators is called
+    event RewardsForAllStakersAndOperatorsCreated(
+        address indexed tokenHopper,
+        uint256 indexed submissionNonce,
+        bytes32 indexed rewardsSubmissionHash,
+        RewardsSubmission rewardsSubmission
+    );
     /// @notice rewardsUpdater is responsible for submiting DistributionRoots, only owner can set rewardsUpdater
     event RewardsUpdaterSet(address indexed oldRewardsUpdater, address indexed newRewardsUpdater);
     event RewardsForAllSubmitterSet(
         address indexed rewardsForAllSubmitter, bool indexed oldValue, bool indexed newValue
     );
+    event TokenHopperSet(address indexed oldTokenHopper, address indexed newTokenHopper);
```

Comment:
seems like an unnecessary / unused event?

Reply:
ahh I added this initially but then realized we can use the rewardsForAll address mapping

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/724/commits/033aa7c464ad36ac98fc0e3440866f024d2fd711